src/routes/incidents/detail/Styles.css

```diff
@@ -7,6 +7,7 @@
     margin-left: 205px;
     margin-top: 60px;
     text-align: left;
+    position:absolute;
```

Comment:
no te voy a mentir, me da miedo esto

Reply:
AH, quedo de un cambio que hice para probar algo chico, ahora lo saco. Igual el absolute no es tan terrible como creiamos. De hecho yo creo que es necesario 

https://www.freecodecamp.org/news/how-to-use-the-position-property-in-css-to-align-elements-d8f49c403a26/